Refrigerator/Program.cs

```diff
@@ -0,0 +1,223 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Remoting.Services;
+using System.Text;
+using System.Threading.Tasks;
+using static Refrigerator.Refrigerator;
+
+namespace Refrigerator
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            List<Refrigerator> refrigerators = new List<Refrigerator>();
+            Refrigerator refrigerator1 = new Refrigerator(1, "Model A", "White", 5);
+            Refrigerator refrigerator2 = new Refrigerator(2, "Model B", "Black", 5);
+            Refrigerator refrigerator3 = new Refrigerator(3, "Model c", "Green", 5);
+
+            refrigerators.Add(refrigerator1);
+            refrigerators.Add(refrigerator2);
+            refrigerators.Add(refrigerator3);
+
+
+            while (true)
+            {
+                Console.WriteLine("Choose an option:");
+                Console.WriteLine("1: List items in the refrigerator");
+                Console.WriteLine("2: Check available space in the refrigerator");
+                Console.WriteLine("3: Add an item to the refrigerator");
+                Console.WriteLine("4: Remove an item from the refrigerator");
+                Console.WriteLine("5: Clean the refrigerator");
+                Console.WriteLine("6: What do I want to eat?");
+                Console.WriteLine("7: List items by expiration date");
+                Console.WriteLine("8: List shelves by available space");
+                Console.WriteLine("9: List refrigerators by available space");
+                Console.WriteLine("10: Prepare for shopping");
+                Console.WriteLine("100: Exit");
+
+                int choice = int.Parse(Console.ReadLine());
+
+                switch (choice)
```

Comment:
switch case is not a good practice. please think on another approach.
if you have any questions about it, write me down here.

thanks